### ISSUE TITLE
[Enhacement] Frontend: display hosts name by fetching api

### ISSUE THREAD

**Issue by infinity1729** [infinity1729/EventListing#21]
**Description**
Currently host/club names associated with listed events are displayed. This was because host names were fetched from `events_api`.
You are required to fetch responses from `hosts_api` and display the names according to that.

**important links**
`http://127.0.0.1:8000/api/hosts_api/`

**Comment by Kumar-Ankit56** [comment]
Please assign me and let me know I Don't backed knowledge with Dango. Could I able to solve it.

**Comment by infinity1729** [comment]
yes! experience with Django isn't required for this issue. API has already been created, you just need to fetch and display the responses.
If you are stuck at some point or need any help do let us know. 😃

**Comment by infinity1729** [comment]
@Kumar-Ankit56  any updates?
Do let us know if you are facing any problem with the code!

**Comment by Kumar-Ankit56** [comment]
@infinity1729  Plz Allow me  some time sem exam is going on
